apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -216,28 +219,71 @@ public partial class SampleRowKeysStream
             }
         }
     }
-    
+
     public partial class BigtableServiceApiClientImpl
     {
         private BigtableServiceApiSettings _defaultSettings;
 
         /// <inheritdoc/>
-        public override BigtableServiceApiSettings DefaultSettings => _defaultSettings;
+        public override string AppProfileId => _defaultSettings.AppProfileId;
+
+        internal override BigtableServiceApiSettings DefaultSettings => _defaultSettings;
```

Comment:
I suspect this can just be

```csharp
internal override BigtableServiceApiSettings DefaultSettings { get; }
```

and assign to it from the constructor, with no expicitly-declared backing field.

Reply:
No, we don't have the constructor here. Only an OnConstruction partial. Same reason why the backing field can't be readonly.

Reply:
Oops - apologies, I'm a muppet.